<launch>
    <arg name="input_global_ref_topic"   default="/global_planning/reference_path" />
    <arg name="input_costmap_topic"      default="/global_planning/costmap" />
    <arg name="input_vehicle_pose_topic" default="/simulation/vehicle_pose" />
    <arg name="input_vehicle_vel_topic"  default="/simulation/vehicle_velocity" />
    <arg name="input_obstacles_topic"    default="/perception_prediction/predicted_obstacles" />
    
    <arg name="output_local_trajectory_topic" default="trajectory" />
    <arg name="output_debug_path_topic"       default="debug/path" />
    <arg name="output_debug_speed_topic"      default="debug/speed_profile" />
    
    
    <node pkg="local_planning" name="local_planning" type="local_planning" output="screen">
        <param name="input_global_ref_topic"   type="str" value="$(arg input_global_ref_topic)" />
        <param name="input_costmap_topic"      type="str" value="$(arg input_costmap_topic)" />
        <param name="input_vehicle_pose_topic" type="str" value="$(arg input_vehicle_pose_topic)" />
        <param name="input_vehicle_vel_topic"  type="str" value="$(arg input_vehicle_vel_topic)" />
        <param name="input_obstacles_topic"    type="str" value="$(arg input_obstacles_topic)" />
        
        <param name="output_local_trajectory_topic" type="str" value="$(arg output_local_trajectory_topic)" />
        <param name="output_debug_path_topic"       type="str" value="$(arg output_debug_path_topic)" />
        <param name="output_debug_speed_topic"      type="str" value="$(arg output_debug_speed_topic)" />
    </node>
    
</launch>